src/LaunchDarkly.EventSource/Resources.resx

```diff
@@ -117,31 +117,19 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="EventSourceService_Read_Timeout" xml:space="preserve">
```

Comment:
I removed the `EventSourceService` from these names because all of these things already by definition apply to stuff in the `LaunchDarkly.EventSource` namespace.